### PR TITLE
Add a bullseye builder

### PR DESCRIPTION
**Pull request by legoktm**
Duplicate the current dockerfiles/ directory into a separate buster
and bullseye directories with the understanding that the buster
one is going to go away pretty soon, so it's not worth abstracting
the logic to be platform agnostic.

I dropped some packages and the timezone step that were already present
in the base `debian:bullseye` image.

I added my PGP key to the trusted set and rebuilt the Cython wheel for Python 3.9,
so now the bootstrap is installable in the bullseye container.

Note: I have not yet pushed the image to quay.io.

### CODE REVIEW

File: build-requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
 build==0.3.0 --hash=sha256:3fe8fac398ae76f534dee92c0db01c2960e271094f8cd52aa9d24a633c244e59
 click==7.1.2 --hash=sha256:3b0769bd65ca00ffcb57975e750fac079639a3d3ce393c5a0d4c65123141c87a
-cython==0.29.22 --hash=sha256:8c9eec7e9de2a30861ca347d0a149cc1482de12fc765fa06c414930e8ce20d0a
+cython==0.29.22 --hash=sha256:8c9eec7e9de2a30861ca347d0a149cc1482de12fc765fa06c414930e8ce20d0a --hash=sha256:df6b83c7a6d1d967ea89a2903e4a931377634a297459652e4551734c48195406
```
Comment (creviera) [inline build-requirements.txt:3]: did this already get diff-reviewed?

**Comment by creviera** [comment]
@legoktm you can add your key to https://github.com/freedomofpress/securedrop-debian-packaging/tree/main/pubkeys

File: build-requirements.txt (hunk 1 above)
Comment (legoktm) [reply build-requirements.txt:3]: It's the hash of the tarball that's already in the repository, so I assume it was reviewed.

**Comment by creviera** [comment]
Does this conflict with https://github.com/freedomofpress/securedrop-debian-packaging/pull/301 in any way?

File: build-requirements.txt (hunk 1 above)
Comment (creviera) [reply build-requirements.txt]: ah, i see that now!

**Review by creviera — APPROVED** [review]
Spoke to @legoktm, did a code walkthrough, and agreed to do more testing (than just building the container locally) post merge so that we can confirm that we can successfully build a workstation component for bullseye. Since we are about to build 9 components, we'll have plenty of opportunity to work out any issues that may arise.

re: https://github.com/freedomofpress/securedrop-debian-packaging/pull/317#issuecomment-1124050120, this shouldn't conflict with https://github.com/freedomofpress/securedrop-debian-packaging/pull/317#issuecomment-1124050120, which adds bullseye build logic for the qubes workstation template.

**Comment by legoktm** [comment]
quay.io/freedomofpress/packaging-debian-bullseye:2022_05_10 has now been pushed.

**Pull request merged**